.github/workflows/deploy.yml

```diff
@@ -38,12 +38,11 @@ jobs:
 
       - name: Build Release And Package
         run: |
-          cargo build --release
+          cargo build --bin parallel-dev --release
```

Comment:
I changed this back because https://github.com/parallel-finance/parallel/actions/runs/959539598 means that github actions have difficuties to build cumulus